### PR TITLE
Added notes about disabling of Node 6 warning message

### PR DESCRIPTION
**Pull request by anatolybolshakov**
Added notes for disabling of Node 6 warning message.

### CODE REVIEW

File: docs/migrateNode10.md [hunk 1]
```diff
@@ -111,4 +111,9 @@ The following `fs` functions all have incompatibilities. In addition, any other
 - fs.unlink
 - fs.utimes
 - fs.write
-- fs.writeFile
\ No newline at end of file
+- fs.writeFile
+
+# How to disable warnings for Node 6 deprecation
```
Comment (alexander-smolyakov) [inline docs/migrateNode10.md:116]: Could you please update the table of content as well?

**Review by EzzhevNikita — APPROVED** [review]

File: docs/migrateNode10.md [hunk 2]
```diff
@@ -111,4 +111,9 @@ The following `fs` functions all have incompatibilities. In addition, any other
...
+To avoid noise in pipeline logs - you can disable Node 6 deprecation warnings by setting up DISABLE_NODE6_DEPRECATION_WARNING agent knob in one of 2 ways:
+- Set up DISABLE_NODE6_DEPRECATION_WARNING pipeline variable as 'true'
+- For self-hosted agents - set up environment variable DISABLE_NODE6_DEPRECATION_WARNING as 'true'
```
Comment (alexander-smolyakov) [inline docs/migrateNode10.md:119]: nit:
```suggestion
- Set up `DISABLE_NODE6_DEPRECATION_WARNING` pipeline variable as `true`
- For self-hosted agents - set up environment variable `DISABLE_NODE6_DEPRECATION_WARNING` as `true` 
```

**Review by alexander-smolyakov — APPROVED** [review]
LGTM and please take a look at the comments.

File: docs/migrateNode10.md (hunk 1 above)
Comment (anatolybolshakov) [reply docs/migrateNode10.md:117]: Updated, thanks!

**Review by alexander-smolyakov — APPROVED** [review]
LGTM, thanks!

**Comment by anatolybolshakov** [comment]
Relevant changes on deployment stage at the moment.

**Pull request merged**